src/raftstore/coprocessor/region_snapshot.rs

```diff
@@ -24,25 +24,32 @@ use raftstore::Result;
 ///
 /// Only data within a region can be accessed.
 pub struct RegionSnapshot {
-    snap: Snapshot,
+    snap: SyncSnapshot,
     region: Region,
```

Comment:
Assume a region has 3 peers, the size is region ID + epoch + 3 * peer + start + end, so the least size may be 72 + start + end. Most of the time, the start and end key may be > 10 bytes, so a region may have 100 bytes, seem not very small.